nameko/amqp/utils.py

```diff
@@ -7,9 +7,6 @@
 from kombu.transport.pyamqp import Transport
 
 
-BAD_CREDENTIALS = (
```

Comment:
if i recall correctly, newer kombu has much better error messages. if so we can probably drop this whole module